### PR TITLE
remove deprecated targets from Makefile

### PR DESCRIPTION
**Pull request by silverwind**
- remove deprecated targets
- ensure 1 empty line between targets

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -181,10 +181,6 @@ errcheck:
 	fi
 	errcheck $(PACKAGES)
 
-.PHONY: lint
-lint:
-	@echo 'make lint is depricated. Use "make revive" if you want to use the old lint tool, or "make golangci-lint" to run a complete code check.'
-
```
Comment (silverwind) [inline Makefile:187]: I'd like to eventually bring this `lint` target back that would run all linters (go,js,css).

**Review by zeripath — APPROVED** [review]

**Comment by silverwind** [comment]
One more change:

- `make clean` now cleans everything, introduced `make clean-go` to just clean build and integration tests.

**Review by sapk — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -128,12 +128,12 @@ node-check:
 		exit 1; \
 	fi
 
-.PHONY: clean-all
-clean-all: clean
+.PHONY: clean
+clean: clean-go
 	rm -rf $(JS_DEST_DIR) $(CSS_DEST_DIR)
 
-.PHONY: clean
```
Comment (6543) [inline Makefile]: this will change expected default behaviour - should we realy change this?

Comment (6543) [reply Makefile]: @silverwind @zeripath @sapk :arrow_up:

Comment (silverwind) [reply Makefile]: It's mostly a personal preference so I can run `make clean build` to refresh all files. If people prefer `clean` to keep only cleaning build/integration, I can remove it again. I think @guillep2k had an opinion on it too.

Also, I'm contemplating whether `clean` should also remove `node_modules`.

Comment (silverwind) [reply Makefile]: If `clean-go` is unneccesary, I'm happy to reduce it to just a single `clean` target.

**Comment by lunny** [comment]
Maybe we should also update the documentation.

File: Makefile (hunk 2 above)
Comment (6543) [reply Makefile]: I normaly use `clean` as it works now ... (I normaly dont touch js code)
-> so **don't** like it if it would be removed completly :/

Comment (silverwind) [reply Makefile]: Reverted it, I guess we can have that discussion another time :)

**Comment by silverwind** [comment]
Reverted controversial `clean` change, should be good to go now.

**Review by 6543 — APPROVED** [review]

File: Makefile [hunk 3]
```diff
@@ -181,10 +181,6 @@ errcheck:
 	fi
 	errcheck $(PACKAGES)
 
-.PHONY: lint
-lint:
```
Comment (guillep2k) [inline Makefile:185]: I'd prefer to make `lint` an alias of `golangci-lint` which I find too long to type.. 😏

File: Makefile (hunk 1 above)
Comment (guillep2k) [reply Makefile:187]: > 
> 
> I'd like to eventually bring this `lint` target back that would run all linters (go,js,css).

Sorry, now I've read this. I'd say go for it! 😄

File: Makefile (hunk 3 above)
Comment (silverwind) [reply Makefile:185]: Yeah, I'm thinking about re-adding `lint` but it would run the JS linter too 😉

**Pull request merged**